synapse/lib/hiveauth.py

```diff
@@ -413,6 +416,32 @@ def reqAuthGate(self, iden):
             raise s_exc.NoSuchAuthGate(iden=iden, mesg=mesg)
         return gate
 
+    def checkUserLimit(self):
+        '''
+        Check if we're at the specified user limit.
+
+        This should be called right before adding/unlocking/unarchiving a user.
+
+        Raises: s_exc_HitLimit if the number of active users is at the maximum.
```

Comment:
got a "_" instead of "." in "s_exc_HitLimit"

Reply:
Fixed. Thanks!